frontend/public/components/nav/admin-nav.tsx

```diff
@@ -85,6 +85,7 @@ const MonitoringNavSection = connect(monitoringNavSectionStateToProps)(Monitorin
 const AdminNav = () => (
   <React.Fragment>
     <NavSection title="Home">
+      <HrefLink href="/dashboards" activePath="/dashboards/" name="Dashboards" required={FLAGS.CAN_LIST_NS} />
```

Comment:
I dont mind adding another flag to required but maybe its not really necessary ? Dashboards still provides some value even if prometheus is not enabled, you have Details, Health, Inventory, Events which do not rely on monitoring.

Reply:
OK is it functional without errors? Then it's probably not needed

Reply:
It works. Here's how dashboars looks like on current master without monitoring

![screencapture-localhost-9000-dashboards-2019-07-19-19_09_55](https://user-images.githubusercontent.com/2078045/61553805-903a5a80-aa5b-11e9-8f40-82d7b68be57a.png)

I have this error in log, but that shouldn't be a problem to fix
```
Warning: Failed prop type: `height` in `VictoryPie` must be a non-negative number.
    in VictoryPie (created by ChartPie)
    in ChartPie (created by ChartDonut)
    in ChartDonut (created by ChartDonutUtilization)
    in ChartDonutUtilization (created by GaugeChart)
    in svg (created by VictoryContainer)
    in div (created by VictoryContainer)
    in VictoryContainer (created by ChartContainer)
    in ChartContainer (created by ChartDonutThreshold)
    in ChartDonutThreshold (created by GaugeChart)
    in Unknown (created by ConnectFunction)
    in ConnectFunction (created by GaugeChart)
    in div (created by ForwardRef)
    in ForwardRef (created by GaugeChart)
    in GaugeChart
    in div
    in Unknown (created by CapacityCard_)
    in div (created by CapacityBody)
    in CapacityBody (created by CapacityCard_)
```

Reply:
We usually hide charts when we know monitoring is not enabled, but that will leave only a couple cards with a big empty space in the middle. I'm not sure how useful this is without Prometheus metrics. Note that this is what native k8s users will see always since we only proxy the Prometheus in `openshift-monitoring` namespace.

Reply:
> We usually hide charts when we know monitoring is not enabled, but that will leave only a couple cards with a big empty space in the middle. I'm not sure how useful this is without Prometheus metrics. Note that this is what native k8s users will see always since we only proxy the Prometheus in `openshift-monitoring` namespace.

Great point. Would we want to potentially "upsell" the use of Prometheus and use empty state designs for these cards? 

I think dropping capacity, utilization, and top consumers is also an option and we could potentially have a slightly different layout where we give Health and Events (Updates will be proposed soon to propose an "Activity" card instead here) more real estate.

FYI @andybraren @yuvalgalanti @mceledonia @rileyhuston